app/src/organisms/Devices/PipetteCard/__tests__/PipetteSettingsSlideout.test.tsx

```diff
@@ -56,24 +61,41 @@ describe('PipetteSettingsSlideout', () => {
         status: 200,
       },
     })
+    mockGetConfig.mockReturnValue({} as any)
+    mockGetAttachedPipetteSettingsFieldsById.mockReturnValue(
+      omit(pipetteSettingsResponseFixture.fakePipetteIdOne.fields, 'quirks')
+    )
     dispatchApiRequest = jest.fn()
     mockUseDispatchApiRequest.mockReturnValue([dispatchApiRequest, ['id']])
-    mockConfigurePipette.mockReturnValue(<div>mock configure pipette</div>)
   })
   afterEach(() => {
     jest.resetAllMocks()
-  })
-  afterEach(() => {
     resetAllWhenMocks()
   })
 
-  it('renders correct text', () => {
-    const { getByText, getByRole } = render(props)
+  it('renders correct heading and number of text boxes', () => {
+    const { getByRole, getAllByRole } = render(props)
+
+    getByRole('heading', { name: 'Left Pipette Settings' })
+    const inputs = getAllByRole('textbox')
+    expect(inputs.length).toBe(9)
+  })
+
+  it('renders close button that calls props.onCloseClick when clicked', () => {
+    const { getByRole } = render(props)
 
-    getByText('Left Pipette Settings')
-    getByText('mock configure pipette')
     const button = getByRole('button', { name: /exit/i })
-    fireEvent.click(button)
+    button.click()
     expect(props.onCloseClick).toHaveBeenCalled()
   })
+
+  it('renders confirm button and calls form onsubmit when clicked', () => {
+    const { getByRole } = render(props)
+
+    const form = getByRole('form', { name: 'configure_pipette_form' })
```

Comment:
There is only one form under test, no need for the `name` filter

```suggestion
    const form = getByRole('form')
```

Reply:
yeah this is weird by test-library has a lot of caveats about which elements can be found by role alone: https://github.com/testing-library/dom-testing-library/issues/474

Reply:
actually, as the issue refers, this is required by the spec, not the library itself

Reply:
This element is not needed for the test. I still think the aria-label should be removed or replaced with something human-readabale